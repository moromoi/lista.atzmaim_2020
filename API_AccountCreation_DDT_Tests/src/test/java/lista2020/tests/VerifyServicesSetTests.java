package lista2020.tests;

import org.approvaltests.Approvals;
import org.testng.annotations.Test;

import java.util.HashSet;

import static org.testng.Assert.assertEquals;

public class VerifyServicesSetTests extends TestBase {

    @Test(priority = 1)
    public void getHolisticJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[1]"));

    }

    @Test(priority = 2)
    public void getConsultansJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[2]"));
    }

    @Test(priority = 3)
    public void getCosmeticsJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[3]"));
    }

    @Test(priority = 4)
    public void getHairSalonJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[4]"));
    }

    @Test(priority = 5)
    public void getInteriorJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[5]"));
    }

    @Test(priority = 6)
    public void getInstallersJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[6]"));
    }

    @Test(priority = 7)
    public void getMakeupJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[7]"));
    }

    @Test(priority = 8)
    public void getNailJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[8]"));
    }

    @Test(priority = 9)
    public void getMassageJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[9]"));
    }

    @Test(priority = 10)
    public void getMoversJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[10]"));
    }

    @Test(priority = 11)
    public void getNutritionistsJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[11]"));
    }

    @Test(priority = 12)
    public void getPestJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[12]"));
    }

    @Test(priority = 13)
    public void getTeachersJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[13]"));
    }

    @Test(priority = 14)
    public void getSalesJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[14]"));
    }

    @Test(priority = 15)
    public void getServiceProvidersJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[15]"));
    }

    @Test(priority = 16)
    public void getPersonalJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[16]"));
    }

    @Test(priority = 17)
    public void getTattooJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[17]"));
    }

    @Test(priority = 18)
    public void getTherapistsJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[18]"));
    }

    @Test(priority = 19)
    public void getAllTypeJSON() {
        for (int i = 0; i < 1; i++) {

            Approvals.verify(app.serviceRestHelper().createAccountWithType("[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18]"));
        }
    }

    @Test(priority = 20)
    public void withoutTypeJSON() {
        for (int i = 0; i < 1; i++) {
            Approvals.verify(app.serviceRestHelper().createAccountWithType("[]"));
        }
    }




    @Test(priority = 21)
    public void deleteAccount() {

        assertEquals(app.serviceRestHelper().deleteAccount(), new HashSet<>() );
    }


}