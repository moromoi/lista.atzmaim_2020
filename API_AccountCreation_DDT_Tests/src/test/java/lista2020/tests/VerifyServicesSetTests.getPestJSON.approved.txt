[{"id":1,"name":"שיחה טלפונית \/ הצעת מחיר","price":"0","duration":30,"color":"#ff98ad","category":{"name":"Pest Controllers","id":1}},{"id":2,"name":"הדברה דירת 2 חדרים","price":"0","duration":60,"color":"#ff98ad","category":{"name":"Pest Controllers","id":1}},{"id":3,"name":"הדברה דירת 3 חדרים","price":"0","duration":60,"color":"#ff98ad","category":{"name":"Pest Controllers","id":1}},{"id":4,"name":"הדברה דירת 4 חדרים","price":"0","duration":90,"color":"#ff98ad","category":{"name":"Pest Controllers","id":1}},{"id":5,"name":"הדברה יחידת דיור","price":"0","duration":60,"color":"#ff98ad","category":{"name":"Pest Controllers","id":1}},{"id":6,"name":"הדברה בית פרטי - קטן","price":"0","duration":120,"color":"#ff98ad","category":{"name":"Pest Controllers","id":1}},{"id":7,"name":"הדברה בית פרטי - גדול","price":"0","duration":180,"color":"#ff98ad","category":{"name":"Pest Controllers","id":1}},{"id":8,"name":"הדברת ביוב מרכזי","price":"0","duration":30,"color":"#ff98ad","category":{"name":"Pest Controllers","id":1}},{"id":9,"name":"הדברה חיצונית \/ חצר","price":"0","duration":60,"color":"#ff98ad","category":{"name":"Pest Controllers","id":1}},{"id":10,"name":"הדברת חנות \/ עסק \/ משרד","price":"0","duration":60,"color":"#ff98ad","category":{"name":"Pest Controllers","id":1}},{"id":11,"name":"הדברת מוסדות ציבור ","price":"0","duration":180,"color":"#ff98ad","category":{"name":"Pest Controllers","id":1}}]